### DISCUSSION TITLE
Anyone notice issues with EPG - I only have six days of data

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#178] [General]
I happened to be looking for some sports events coming up and noticed that I only have data up to 7pm on Dec 22.  Normally I would have about two weeks worth of data.  I don't even know where to look for any logs.

I know that I have a valid SD account as I recently renewed it.

Anyone else noticing an issue like this?  I hope that something isn't broken in the EPG loading in SageTV.

**Comment by wayner9** [comment]
I just realized that I changed my password at Schedules Direct when I paid for the next year.  Does anyone remember where you change this in SageTV?

**Comment by TGsell1** [comment]
It’s in the video sources setup.  Look at the lineups and there should be some reference to SD and hence a log on.   I’m not at my server or I would give better guidance Sent from my iPhoneOn Dec 16, 2024, at 10:55 AM, wayner9 ***@***.***> wrote:﻿
I just realized that I changed my password at Schedules Direct when I paid for the next year.  Does anyone remember where you change this in SageTV?

**Reply by wayner9** [reply]
Thanks, I will check that out when I get home tonight.  I wonder if I have to change it in all lineups that I have?  Or if I even have to go through the source/channel setup again.

**Comment by Ugadata** [comment]
You will need to go through the motions of making some kind of line up change.  But you can change to the same as you already have or "exit out" once the password is changed.  

Keep in mind, depending on your Sage settings, when typing the password, it maybe like typing on an old telephone/cellphone.

![image_2024-12-16_121119299](https://github.com/user-attachments/assets/af9f8c45-1d2c-4dc8-a3c0-d4e1a0bc7a7c)

**Reply by wayner9** [reply]
Thanks - it seemed to work if I changed the username and password on this screen and then backed out.

Thanks for the help everyone.

I imagine that the EPG breaking is a big risk since I don't think there is anyone around who could fix it if that happens.
